admin_guide/limits.adoc

```diff
@@ -181,6 +210,98 @@ Across all containers in a pod, the following must hold true:
 |===
 // end::admin_limits_pod_limits[]
 
+[[image-limits]]
+=== Image Limits
+
+// tag::admin_limits_image_limits[]
+
+*Supported Resources:*
+
+* Storage
+
+*Resource type name:*
+
+- `openshift.io/Image`
+
+[cols="3a,8a",options="header"]
+|===
+|Constraint |Behavior
+
+|`*Max*`
+|`image.dockerImageMetadata.size` less than or equal to Max[resource]
```

Comment:
I wasn't sure either. The other field names are single word items. I'll lower it.